tests/core/engines/alpha/steps/engines.py

```diff
@@ -140,3 +140,30 @@ def when_messages_are_emitted(
     assert all(e is not None for e in message_event_generator_results[0].events)
 
     return list(cast(list[EmittedEvent], message_event_generator_results[0].events))
+
+
+@step(when, "uttring actions", target_fixture="emitted_events")
+def when_uttring_actions(
```

Comment:
uttering

Reply:
No need "uttring actions"
@step(when, "uttering is triggered", ...
def when_uttering_is_triggered